README.md

```diff
@@ -161,11 +161,7 @@ wss.broadcast = function broadcast(data) {
 wss.on('connection', function connection(ws) {
   ws.on('message', function incoming(data) {
     // Broadcast to everyone else.
-    wss.clients.forEach(function each(client) {
-      if (client !== ws && client.readyState === WebSocket.OPEN) {
-        client.send(data);
-      }
-    });
+    wss.broadcast(data);
```

Comment:
This is not the same as it broadcasts to **all** clients without excluding the one that received the message.